oci/layout/oci_transport.go

```diff
@@ -36,22 +36,21 @@ func (t ociTransport) ParseReference(reference string) (types.ImageReference, er
 	return ParseReference(reference)
 }
 
-var refRegexp = regexp.MustCompile(`^([A-Za-z0-9._-]+)+$`)
+var refRegexp = regexp.MustCompile(`^([A-Za-z0-9:._-]+)+$`)
```

Comment:
Do we want to explicitly mirror the spec at https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys here?
(@runcom?)